### DISCUSSION TITLE
Build does not support openssl-3

### DISCUSSION THREAD

**Discussion by alerque** [OpenIDC/mod_auth_openidc#967] [General]
Building Arch Linux packages for this with the latest 2.4.12.1 release I noticed it does not build with the current OpenSSL 3.0.7 release. I had to use legacy support packages with v1.1 to build.

**Comment by zandbelt** [comment]
@alerque could this be caused by cjose 0.6.1? If so, please upgrade to the maintenance fork at https://github.com/zmartzone/cjose/ as suggested in https://github.com/cisco/cjose/issues/121

**Reply by alerque** [reply]
Yes it could be related. I'll try to get `cjose` moved over to the fork you link and then try building with that. I'll report back when that update comes through packaging if that was the problem or not. I'm not the maintainer of the cjose package though so it might take a little while to figure out.

**Reply by alerque** [reply]
Confirmed after getting Arch packages switched to the cjose fork you linked I'm now able to build this project with openssl-3. Thanks for the pointer!
